samples/js/flickr/find-photos/src/main.js

```diff
@@ -93,8 +100,20 @@ async function main(): Promise<void> {
         title: v.title,
         tags: new Set(v.tags ? v.tags.split(' ') : []),
         sizes: getSizes(v),
+        datePublished: newStruct('Date', {
```

Comment:
It's not different? I'm referring to making sure that
- struct is called 'Date'
- property is nsSinceEpoch

```
newDateStruct(Number(v.dateupload) * nsInSecond)
...
newDateStruct(Number(v.lastupdate) * nsInSecond)
...
newDateStruct(Date.parse(v.datetaken) * nsInMillisecond))
```